src/App.js

```diff
@@ -1,38 +1,33 @@
 // @flow
 
 import React from 'react';
+import DAOApi from 'brewskey.js-api';
 import { StyleSheet, Text, View } from 'react-native';
-import { observer } from 'mobx-react';
+import { observer, Provider } from 'mobx-react';
 import { action, observable } from 'mobx';
 import { Button } from 'react-native-elements';
+import config from './config';
+import RootStore from './stores/RootStore';
+import RootRouter from './routes';
 
-@observer
-export default class App extends React.Component {
-  @observable counter = 0;
-
-  @action increment = () => this.counter++;
+DAOApi.initializeDAOApi({
+  endpoint: `${config.host}api/v2/`,
+});
 
-  componentDidMount() {
-    setInterval(this.increment, 1000);
-  }
+const rootStore = new RootStore();
+let stores = {};
+Object.getOwnPropertyNames(new RootStore()).map(
+  (storeName: string): Object => (stores[storeName] = rootStore[storeName]),
+);
 
+class App extends React.Component {
   render() {
     return (
-      <View style={styles.container}>
-        <Text>Brewskey supa dupa app</Text>
-        <Text>Counter: {this.counter}</Text>
-        <Button raised title="Test Button" />
-      </View>
+      <Provider {...stores}>
```

Comment:
I can:
1) do like i did here, and use `inject('storeName')` in containers.
2) set rootStore to provider, and use `inject(({ rootStore }) => ({ authStore: rootStore.authStore }))` in cosinntainers
3) don't use provider at all, make singeltons and use imports all over the files.